GitUI/BranchTreePanel/BaseRefTree.cs

```diff
@@ -80,5 +84,51 @@ internal override void UpdateVisibility()
 
             base.UpdateVisibility();
         }
+
+        protected IEnumerable<IGitRef> PriotizedBranches(IReadOnlyList<IGitRef> branches)
```

Comment:
```suggestion
        protected IEnumerable<IGitRef> PrioritizedBranches(IReadOnlyList<IGitRef> branches)
```
plus a few more occurrences including the settings page.

Reply:
Cannot spell Prioritize....